artemis-distribution/src/main/assembly/dep.xml

```diff
@@ -86,6 +86,7 @@
             <include>org.apache.commons:commons-dbcp2</include>
             <include>org.apache.commons:commons-pool2</include>
             <include>io.netty:netty-all</include>
+            <include>io.netty.incubator:netty-incubator-transport-native-io_uring</include>
```

Comment:
I similarly think its premature to distribute it.

Reply:
I agree, leave it on a branch till there is a non incubator netty dependency.